frontend2/src/contexts/wallet.tsx

```diff
@@ -11,19 +11,21 @@ import { FewchaWallet } from "fewcha-plugin-wallet-adapter";
 import { PetraWallet } from "petra-plugin-wallet-adapter";
 import { create } from "zustand";
 
+import { SupportedNetworkName } from "@/components/Canvas/types";
```

Comment:
It might be better to define this type in this file since it's a more central location and the network name isn't really coupled with the canvas